app/src/main/java/com/example/fittracker/MainActivityModel.java

```diff
@@ -0,0 +1,18 @@
+package com.example.fittracker;
+
+public class MainActivityModel implements LogInContract.Model{
+    private UserBase listUsuarios;
+    public MainActivityModel (UserBase listUsuarios){
+        this.listUsuarios = listUsuarios;
+    }
+
+    @Override
+    public boolean isValidLogIn(String mail, String password) {
```

Comment:
this parametres are not necessary, because you will have here the correct simulated values for the login